hmt_escrow_js/ETHInterface.js

```diff
@@ -0,0 +1,146 @@
+require('dotenv').config()
+
+const EscrowABI= require('./ABIs/Escrow')
+const EscrowFactoryABI = require('./ABIs/EscrowFactory')
+const HMTTokenInterfaceABI = require('./ABIs/HMTTokenInterface')
+
+const HDWalletProvider = require("truffle-hdwallet-provider")
+const web3 = require('web3')
+const isUrl = require('is-url')
+
+let HMTOKEN_ADDR = process.env.HMTOKEN_ADDR || "0x4C18A2E51edC5043e9c4B6b0757990A4Ac13797f"
+
+class ETHInterface {
+  constructor(MNEMONIC, INFURA_KEY, NETWORK) {
+    // Setups Required
+    if (!MNEMONIC || !INFURA_KEY || !NETWORK ) {
+      console.error("Please set a mnemonic, infura key,and network")
+      return
+    }
+    const provider = new HDWalletProvider(MNEMONIC, `https://${NETWORK}.infura.io/v3/${INFURA_KEY}`)
```

Comment:
You should also allow for the option to connect to Infura via Websocket, I recently had an issue with that, meaning I couldn't query the contract for events, since those RPC calls are available only for Websocket, not HTTPS. But this is really just a note for the future, no need to do that now.

Reply:
That's weird, over this the event calls were coming in just fine? Added a note

Reply:
Hmm, I can't see any note, would you point to me to where you added that?